eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Ratify.hs

```diff
@@ -199,28 +198,29 @@ spoAccepted re rs gas =
 -- vote are considered as `Abstain` votes.
 spoAcceptedRatio :: forall era. RatifyEnv era -> GovActionState era -> Rational
 spoAcceptedRatio
```

Comment:
As we've discussed on the call, if you can make this function look a lot more like `drepAcceptedRatio` this PR will be good to go